CefSharp.Core/Internals/CefPostDataWrapper.cpp

```diff
@@ -0,0 +1,96 @@
+#include "Stdafx.h"
```

Comment:
Can you merge this back into the `.h` file? I know that sounds like a strange request, It just makes comparing your changes so much easier.